lasagne/tests/layers/test_normalization.py

```diff
@@ -50,8 +50,8 @@ def ground_truth_normalizer(c01b, k, n, alpha, beta):
         for c in range(out.shape[2]):
             for x in range(out.shape[3]):
                 out[:, r, c, x] = ground_truth_normalize_row(
-                        row=c01b[:, r, c, x],
-                        k=k, n=n, alpha=alpha, beta=beta)
+                    row=c01b[:, r, c, x],
+                    k=k, n=n, alpha=alpha, beta=beta)
```

Comment:
I have actually seen the same PEP8 error on some machines but not others. It doesn't occur on my laptop for example, where I do most of my development (nor on Travis). It might be related to what version of pep8 is installed.